docs/tutorials/see-all-tutorials.md

```diff
@@ -11,6 +11,7 @@
 
 * [Key value database](../application-tutorial/key_value_database.ipynb)
```

Comment:
hum, @umut-sahin , aren't we supposed to have all tutorials in the same directory, in concrete/frontends/concrete-python/examples? KV and SHA256 are not there

Reply:
I did https://github.com/zama-ai/concrete-internal/issues/814